src/styles/header.scss

```
@@ -0,0 +1,63 @@
+@import "variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 50px;
+  padding-right: 50px;
+  height: 60px;
+  
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  top: 10px;
+  left: 50px;
+}
+
+.menu_list {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+}
+
+.menu_item {
+  list-style-type: none;
+}
+
+.menu_link {
```

```suggestion
.menu {
  &_list {
    display: flex;
    justify-content: space-between;
    margin: 0;
    padding: 0;
  }

  &_item {
    list-style-type: none;
  }
}
```
это вложеность ее нужно использовать